src/ProjectTeam.php

```diff
@@ -59,6 +59,7 @@ class ProjectTeam extends CommonDBRelation
     public static $items_id_2          = 'items_id';
     public static $checkItem_2_Rights  = self::DONT_CHECK_ITEM_RIGHTS;
 
+    /** @var class-string<CommonDBTM>[] */
     public static $available_types     = ['User', 'Group', 'Supplier', 'Contact'];
```

Comment:
Not directly related, but can be fixed here I guess
```suggestion
    public static $available_types     = [User::class, Group::class, Supplier::class, Contact::class];
```